libs/auth/src/components/account.component.html

```diff
@@ -0,0 +1,50 @@
+<button
+  *ngIf="account.id !== specialAccountAddId; else addAccount"
```

Comment:
We try to avoid using these conditionals in templates.
```suggestion
  *ngIf="account.id !== specialAccountAddId"
```